src/Sylius/Bundle/ResourceBundle/Routing/ResourceLoader.php

```diff
@@ -169,6 +169,9 @@ private function createRoute(MetadataInterface $metadata, array $configuration,
         if (!empty($configuration['criteria'])) {
             $defaults['_sylius']['criteria'] = $configuration['criteria'];
         }
+        if (!empty($configuration['filterable'])) {
```

Comment:
Understood. If so, wouldn't removing the default value `false` from the routing configuration node make it behave like before but with the added possibility to customise it? If not set at all it won't even set that `filterable` setting in `ResourceLoader` so it will fall back to the `RequestConfiguration` defaults.